### ISSUE TITLE
TypeVar passed name and assigned name must match

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#1667]
# Rule request

## Thesis

```python
from typing import TypeVar

# bad
MyType = TypeVar('AnotherName')
# good
MyType = TypeVar('MyType')
```

## Reasoning

Be consistent between what you see in the code and what MyPy outputs in the console.

**Comment by sobolevn** [comment]
Why should we? If that is covered by mypy? Can you please explain?

**Comment by orsinium** [comment]
Right, it is. Sorry, I use another type checker.
